kb/cryptography/providing-diffie-hellman-or-dhm-parameters.md

```diff
@@ -8,10 +8,6 @@ Developers have the option to set the DHM parameters for SSL servers with `mbedt
 
```

Comment:
This whole document is no longer applicable after 3.x, so we should just put a note at the beginning saying so.